vaas/vaas/settings/base.py

```diff
@@ -78,6 +78,9 @@ def serialize(value: any) -> str:
     'simple_history.middleware.HistoryRequestMiddleware',
 ]
 
+INSTALLED_APPS = tuple(INSTALLED_PLUGINS) + INSTALLED_APPS
```

Comment:
It seems that INSTALLED_PLUGINS is redundant
```
INSTALLED_APPS =tuple(env.json('INSTALLED_PLUGINS', default=[])) + INSTALLED_APPS
```
Similarly, MIDDLEWARE_PLUGINS could be eliminated:
```
MIDDLEWARE += env.json('MIDDLEWARE_PLUGINS', default=[])
```